examples/submit_custom_vocabularies.js

```diff
@@ -46,6 +46,12 @@ const token = require('./config/config.json').access_token;
             , details: ${cv_submission.failure_detail}`)
     }
 
+    /**
+     * Gets list of custom vocabularies
```

Comment:
```suggestion
     * Gets list of custom vocabularies' information
```

Trying to clarify that this gets the information here, just like we are clear about it in the client method name.